File: agents/cwe/tests/fixtures/cwe778/langs/swallow.rs
fn a(p: &str) -> Result<(), E> {
    let f = read(p)?;                       // propagates - MUST NOT fire
    let g = read(p).unwrap();               // panics loudly - MUST NOT fire
    let h = read(p).expect("needed");       // panics loudly - MUST NOT fire
    if let Err(e) = write(p) {
        return;
    }
    if let Err(e) = write(p) {
        tracing::error!("write failed: {e}");
    }
    match write(p) {
        Ok(v) => v,
        Err(e) => {
            0
        }
    }
    write(p).ok();
    Ok(())
}

File: agents/cwe/tests/fixtures/cwe778/langs/swallow.swift
func a(p: String) {
    do { try write(p) } catch { }
    do { try write(p) } catch { os_log("write failed: %@", "\(error)") }
    do { try write(p) } catch let e as IOError { }
    do { try write(p) } catch let e as IOError { NSLog("io: \(e)") }
    do { try write(p) } catch { throw e }
    let v = try? write(p)
    let w = try! write(p)
}
